internal/pkg/selectors/selectors.go

```diff
@@ -154,3 +157,28 @@ func GetNoneSelector(inst *unstructured.Unstructured) (bool, error) {
 	}
 	return noneSelector, nil
 }
+
+var resourceExclusions = []string{"istio-ca-root-cert", "kube-root-ca.crt"}
+
+// CheckForExclusion returns true to indicate that this object is excluded from being propagated
```

Comment:
Maybe `checkForExclusions` (plural) is `isExcluded`? Also make it private until we have some reason to want to export this function.

Reply:
have now updated this method to isExcluded()